### ISSUE TITLE
User and password in R::connect()

### ISSUE THREAD

**Issue by rinopazzo** [AtnNn/librethinkdbxx#35]
Is there a way to specify user and password in R::connect()?

**Comment by AtnNn** [comment]
No, that feature is not implemented. This driver supports the old `auth` field.

**Comment by TotalKrill** [comment]
Is there any example of how to use the 'auth' field, I am probably missing something.

The driver still works great btw!

Edit: For anyone else that wonders. The old 'auth' field will default to the password of the 'admin' account

so connecting with R::connect("address", 28015, "PasswordOfAdminAccount") will still work, but the docs does not specify for how long since this may be deprecated. 

https://rethinkdb.com/docs/writing-drivers/
